e2e/rbd.go

```diff
@@ -38,6 +38,11 @@ var (
 	rbdTopologyDataPool = "replicapool" // NOTE: should be different than rbdTopologyPool for test to be effective
 )
 
+const (
```

Comment:
move this to utils? and add a comment for this one what these variables are and when it can be used?